framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -267,9 +257,10 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
     */
 
     /**
-     * Returns the boundaries of the pane.
+     * Returns the dimensions of the pane.
      *
-     * @return {Map} The pane boundaries.
+     * @return {{width: number, height: number}|null} The pane dimension in pixel. Contains
```

Comment:
i'll go with this unless it breaks the API viewer or jsdoc linting - I've often wished for a decent way to describe map members